compiler/src/dotty/tools/dotc/core/Definitions.scala

```diff
@@ -1858,24 +1879,6 @@ class Definitions {
   def isContextFunctionType(tp: Type)(using Context): Boolean =
     asContextFunctionType(tp).exists
 
-  /** An extractor for context function types `As ?=> B`, possibly with
-   *  dependent refinements. Optionally returns a triple consisting of the argument
-   *  types `As`, the result type `B` and a whether the type is an erased context function.
-   */
-  object ContextFunctionType:
```

Comment:
Isn't it still nice for convenience to keep an extractor just for context functions?